src/routes/home/components/CustomDisciplineDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import { useNavigation } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React from 'react'
+import styled from 'styled-components/native'
+
+import Button from '../../../components/Button'
+import { BUTTONS_THEME } from '../../../constants/data'
+import { Discipline } from '../../../constants/endpoints'
+import labels from '../../../constants/labels.json'
+import { RoutesParams } from '../../../navigation/NavigationTypes'
+import { childrenLabel } from '../../../services/helpers'
+import { ButtonContainer, NumberText, UnitText } from './DisciplineCard'
+
+const TextContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: ${props => props.theme.spacings.sm} 0 ${props => props.theme.spacings.xs};
+`
+
+interface PropsType {
+  discipline: Discipline
+}
+
+const CustomDisciplineDetails = ({ discipline }: PropsType): JSX.Element => {
+  const navigation = useNavigation<StackNavigationProp<RoutesParams, 'Home'>>()
+
+  const navigate = (): void => {
+    navigation.navigate('DisciplineSelection', {
+      discipline
+    })
```

Comment:
Why no progress saving/displaying for custom disciplines? What is the difference?

Reply:
Custom disciplines do not necessarily have a logical order, e.g. structur can be 
- School xy
  - German
    - Class 12a
    - Class 12b

So there is no possibility to suggest the next module to the user. Progress is not displayed as in the designs, because there is no use of showing a user (s)he has done 10% of the modules of the school because 100% is not the aim. 

Saving progress of custom disciplines and show it in DisciplineSelectionScreen and ExercisesScreen makes sense, I will create a ticket for it.

Reply:
Thanks for explaining